src/rexi/src/rexi_sup.erl

```diff
@@ -22,6 +22,9 @@ start_link(Args) ->
 init([]) ->
     {ok,
         {{rest_for_one, 3, 10}, [
+            % TODO: Compatibility clause rexi_server. This was used by
+            % server_per_node=false mode once After 3.4+ releases, this can be
```

Comment:
I noticed that you used `> 3.4` in the previous comment, and wondered if one form is preferred, or if you feel that they mean different things?

Reply:
Good point. That is silly. I'll use 3.4+ for consistency.

Reply:
Updated. Made both 3.4+.